### ISSUE TITLE
Tests interrogating self.client.get() return value's context member triggers "Instance of 'WSGIRequest' has no 'context' member (no-member)"

### ISSUE THREAD

**Issue by craiga** [PyCQA/pylint-django#78]
The following test code:

```
response = self.client.get(self.url)
self.assertEqual(response.context['object'], expected_object)
```

…causes the following pylint message:

```
E:256,52: Instance of 'WSGIRequest' has no 'context' member (no-member)
```

pylint==1.5.6 and pylint-django==0.7.2.

**Comment by craiga** [comment]
I wrote a test case for this, but haven't had time to fix it.

```
"""
Checks that Pylint does not complain about a standard test.
"""

from django.test import TestCase
from django.db import models


class SomeModel(models.Model):
    """Just a model."""
    pass


class SomeTestCase(TestCase):
    """A test cast."""
    def test_thing(self):
        """Test a thing."""
        expected_object = SomeModel()
        response = self.client.get('/get/some/thing/')
        self.assertEqual(response.status_code, 200)
        self.assertEqual(response.context['object'], expected_object)

```

**Comment by raymondberg** [comment]
I just ran into this today. Would love a fix!

**Comment by jaslys** [comment]
I fix that error by installing the pylint plugin ! Just run the following command
